frontend/src/components/advanced-auto-filter/advanced-auto-filter.tsx

```diff
@@ -194,40 +177,41 @@ const AdvancedAutoFilter: FC = () => {
               list={raceRange.map((item: number) => item.toString())}
               minTitle="Km Min"
               maxTitle="Km Max"
-              minFilterName={FiltersNames.RACE_START}
-              maxFilterName={FiltersNames.RACE_END}
-              selectedMin={filters.raceStart}
-              selectedMax={filters.raceEnd}
+              rangeName={RangeNames.RACE}
+              minFilterName={RangeValueNames.RACE_START}
+              maxFilterName={RangeValueNames.RACE_END}
+              selectedMin={rangeFilters.race.raceStart}
+              selectedMax={rangeFilters.race.raceEnd}
               onChange={handleRangeChange}
             />
           </div>
           <CheckboxList
             title="Color"
             list={options && options.colors}
-            checkedList={filters.colorId}
+            checkedList={checkLists.colorId}
             onListCheck={handleCheckboxListChange}
-            filterName={FiltersNames.COLOR_ID}
+            filterName={CheckListsNames.COLOR_ID}
```

Comment:
It's because of difficulties of showing and removeing every type of filter from "applied filters bar". Checklists the easiest of all, we have id's to every checked option. But in ranges there aren't id's. I can reach every range only by it's name. Or other approach - to put id in every range. I already does it in Brand Details - id: Date.now().toString()